content/guides/tutorials/dnn-overview/index.md

```diff
@@ -42,4 +42,4 @@ The DNN Documentation Center is organized based on these persona groups:
 *   [Developers](http://dnndocs.com/content/developers/)
 *   [Designers](http://dnndocs.com/content/designers/)
 *   [Content Managers](http://dnndocs.com/content/content-managers/) includes content managers, editors, and creators.
-*   [Community Managers](http://dnndocs.com/content/community-managers/)
+*   Community Managers
```

Comment:
Community Managers, as an Evoq-only feature, has been removed from the docs. So just remove the link and text entirely.

Reply:
Resolved in a new push just now.  :)